### PR TITLE
[PW-10478] Quality stability slider

### PR DESCRIPTION
**Pull request by bitmovin-kenny**
Since we now have a way to influence the players behavior when it comes to choosing the quality in LL environments, we decided to update the LL demo to include it.

BigScreen:
image

Smaller Screen:
image

### CODE REVIEW

File: player/low-latency-streaming/js/script.js [hunk 1]
```diff
@@ -50,10 +63,7 @@ var conf = {
         muted: true,
     },
     adaptation: {
-        logic: 'low-latency-v1',
```
Comment (offnertho) [inline player/low-latency-streaming/js/script.js:53]: why did you remove this. we need this.

File: player/low-latency-streaming/js/script.js [hunk 2]
```diff
@@ -50,10 +63,7 @@ var conf = {
...
         preload: false,
-        // Encourage switching to a higher quality sooner
-        qualityStabilityBalance: 0.3,
```
Comment (offnertho) [inline player/low-latency-streaming/js/script.js:56]: is this set on startup?

**Review by offnertho — CHANGES_REQUESTED** [review]

File: player/low-latency-streaming/js/script.js (hunk 1 above)
Comment (bitmovin-kenny) [reply player/low-latency-streaming/js/script.js:53]: That was an accident caused by merging, nice catch will revert.

**Review by offnertho — APPROVED** [review]
